scripts/gceworker.sh

```diff
@@ -99,7 +102,15 @@ case "${cmd}" in
     start_and_wait "${NAME}"
 
     gcloud compute scp --recurse "build/bootstrap" "${NAME}:bootstrap"
-    gcloud compute ssh "${NAME}" --ssh-flag="-A" --command="./bootstrap/bootstrap-debian.sh"
+    gcloud compute ssh "${NAME}" --ssh-flag="-A" --command="DISABLE_COCKROACHDB=$DISABLE_COCKROACHDB DISABLE_MANAGED_SERVICE=$DISABLE_MANAGED_SERVICE ./bootstrap/bootstrap-debian.sh"
+
+    # Copy the ssh keys to the gceworker, this is needed to clone the managed-service repo
+    echo -n "Enter the path to the ssh key [default ~/.ssh]: "
```

Comment:
Making the message clearer would be an improvement. What SSH key am I supposed to use? What will it be used for?

What if I don't want to copy any of my SSH keys? If I don't want to clone `managed-service`, copying it isn't necessary.

Reply:
Given the message, people will assume that you can input something like `~` and it will be expanded to the home directory. Is that correct? Did you verify that that works?

Reply:
`~` will not expand to home dir. I am updating the text. Thanks for pointing out.